tests/unit/test_datasets_downloader.py

```diff
@@ -1,10 +1,13 @@
-import asyncio
 import os
 from unittest import TestCase
 from unittest.mock import Mock, patch
 
+import asyncio
+
+import aiohttp
 from aiohttp.test_utils import make_mocked_request
 from aiohttp.web import Response
+from aiohttp.client_exceptions import TimeoutError
```

Comment:
Maybe we have too many imports from `aiohttp`. Either you keep what the module was originaly using (e.g. `from aiohttp import ClientSession` to access `ClientSession` – instead of `import aiohttp` and then `aiohttp.ClientSession`) **or** you remove extra imports and access everything from the main `aiohttp` module.